addons/binding/org.openhab.binding.leapmotion/README.md

```diff
@@ -0,0 +1,54 @@
+# LeapMotion Binding
+
+The LeapMotion controller is a gesture sensoring device that uses stereoscopic cameras and is connected through USB.
```

Comment:
* Can you also add a link to the [Leap Motion](https://www.leapmotion.com/) website here so it's easy to find more information about the company/device?
* On the website they refer to themselves as "Leap Motion" and not "LeapMotion" so you might want to update all references for consistency.
* Please place each sentence on a new line in this Markdown file (where possible) so it's easier to add review comments.
* Can you also fix the remaining (minor) SAT issues? :-)

I think this is the first binding that uses profiles so it makes a good example! :+1: